compiler/source-resolver/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@noir-lang/noir-source-resolver",
+  "name": "@noir-lang/source-resolver",
   "version": "1.1.4",
```

Comment:
Isn't this version supposed to be updated if you want versions to be synced?

Reply:
I added this package to the release-please-config, so once this gets merged, it should update in the release-please PR